instrumentation/couchbase/couchbase-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/couchbase/v2_0/CouchbaseInstrumentationModule.java

```diff
@@ -32,6 +32,11 @@ public List<TypeInstrumentation> typeInstrumentations() {
     return asList(new CouchbaseBucketInstrumentation(), new CouchbaseClusterInstrumentation());
   }
 
+  @Override
+  public String getModuleGroup() {
+    return "couchbase";
+  }
```

Comment:
should we add the `test-indy` label to this PR?

Reply:
now indy tests are run by default, that is how I noticed that they failed

Reply:
cool, good to know, thanks!